src/index.ts

```diff
@@ -68,14 +79,10 @@ export default class IncludeFragmentElement extends HTMLElement {
 
   constructor() {
     super()
-    // eslint-disable-next-line github/no-inner-html
-    this.attachShadow({mode: 'open'}).innerHTML = `
-      <style> 
-        :host {
-          display: block;
-        }
-      </style>
-      <slot></slot>`
+    const shadowRoot = this.attachShadow({mode: 'open'})
+    const style = shadowRoot.appendChild(document.createElement('style'))
+    style.textContent = `:host {display: block;}`
+    style.appendChild(document.createElement('slot'))
```

Comment:
Is this a safe rewrite?

Reply:
```suggestion
    const shadowRoot = this.attachShadow({mode: 'open'})
    const style = document.createElement('style'))
    style.textContent = `:host {display: block;}`
   shadowRoot.append(style. document.createElement('slot'))
```

Reply:
Uh, GithHub mobile appears to be buggy about accepting suggestions, sorry if you got confusing notification about this thread.